docs/book/src/tasks/experimental-features/cluster-class/operate-cluster.md

```diff
@@ -42,3 +52,96 @@ kubeadmcontrolplane.controlplane.cluster.x-k8s.io/clusterclass-quickstart-XXXX
+```
+After a minute the MachineDeployment will have scaled down to 1 replica:
+
+```bash
+NAME                         CLUSTER           REPLICAS   READY   UPDATED   UNAVAILABLE   PHASE     AGE   VERSION
+capi-quickstart-md-0-d9rxm   capi-quickstart   1          1       1         0             Running   25m   v1.23.3
+```
+
+## Scale a ControlPlane
+
+A similar process to that used to [scale a MachineDeployment](#scale-a-machinedeployment) can be used to scale a ControlPlane.
+
+This is done by changing the ControlPlane replicase field at `/spec/topology/controlPlane/replica` in the Cluster object. The command is:
+
+```bash 
+kubectl  patch cluster capi-quickstart --type json --patch '[{"op": "replace", "path": "/spec/topology/controlPlane/replicas",  "value": 1}]'
+```
+
+## Add a MachineDeployment
+Assume we have created a CAPD cluster with ClusterClass and Kubernetes v1.23.3 (as documented in the [Quick Start guide]). Initially we should have a single MachineDeployment with 3 replicas. Running
+```bash 
+kubectl get machinedeployments
+```
+Will give us:
+```bash 
+NAME                                                            CLUSTER           REPLICAS   READY   UPDATED   UNAVAILABLE   PHASE     AGE   VERSION
+machinedeployment.cluster.x-k8s.io/capi-quickstart-md-0-XXXX   capi-quickstart   3          3       3         0             Running   21m   v1.23.3
+```
+
+A new MachineDeployment can be added to the Cluster by adding a new MachineDeployment spec under `/spec/topology/workers/machineDeployments/`. To do so we can patch our Cluster with:
+```bash 
+kubectl  patch cluster capi-quickstart --type json --patch '[{"op": "add", "path": "/spec/topology/workers/machineDeployments/-",  "value": {"name": "second-deployment", "replicas": 1, "class": "default-worker"} }]'
+```
+
+After a minute to scale the new MachineDeployment we get:
+```bash
+NAME                                      CLUSTER           REPLICAS   READY   UPDATED   UNAVAILABLE   PHASE     AGE   VERSION
+capi-quickstart-md-0-d9rxm                capi-quickstart   1          1       1         0             Running   39m   v1.23.3
+capi-quickstart-second-deployment-d77h6   capi-quickstart   1          1       1         0             Running   99s   v1.23.3
+```
+Our second deployment uses the same underlying MachineDeployment Class as our initial deployment. In this case they will both have the exact same underlying machine templates. In order to modify the templates MachineDeployments are based on take a look at [Changing a ClusterClass].
+
+## Rebase a Cluster
+To perform more significant changes using a Cluster as a single point of control, it may be necessary to change the ClusterClass that the Cluster is based on. This is done by changing the class referenced in `/spec/topology/class`.
+
+To read more about changing an underlying class please refer to [ClusterClass rebase]
+
+## Use variables
```

Comment:
Short intro about the effect of changing variables, that is achieving a slightly different variant of Cluster topology via ClusterClass patches